packages/script-editor/src/ScriptEditor.tsx

```diff
@@ -80,7 +72,8 @@ export class ScriptEditor extends DocumentWidget<
   private runDisabled: boolean;
   private kernelSelectorRef: RefObject<ISelect> | null;
   private controller: ScriptEditorController;
-  private editorLanguage: string;
+  abstract getLanguage(): string;
+  abstract getIcon(): LabIcon | string;
```

Comment:
why are we passing `labIcon` or `string`? we do have an rIcon defined in ui-components and thus I was expecting to use that in favor of consistency.

Reply:
I believe our `rIcon` is created differently, that's why we define it with a string, I didn't change the original implementation for it. 
When I import it from `ui-components` library the same way we add python icons, the r icon isn't loaded properly in the launcher, file browser or editor tab:
![image](https://user-images.githubusercontent.com/25207344/150416202-934b4410-ed2f-46e5-9bb1-948133334104.png)

@ajbozarth might be able to elaborate on why/how that works?

Reply:
I can't remember why this fix works, but there's an issue with LabIcons where depending on the DOM within a svg added multiple copies of it causes problems. The R icon is the icon we found that has this issue.